source/MapDetailPanel.cpp

```diff
@@ -612,14 +650,37 @@ void MapDetailPanel::DrawInfo()
 		PointerShader::Draw(uiPoint + Point(0., 20.), Point(1., 0.),
 			10.f, 10.f, 0.f, medium);
 
-	const double relativeTradeY = mapInterface->GetValue("relative trade Y after planet");
-	uiPoint = Point(Screen::Left() + startingX, edgePos.Y() + relativeTradeY);
+	// Edges:
+	Point pos(Screen::Left(), Screen::Top());
+	const Sprite *bottom = SpriteSet::Get("ui/bottom edge");
+	Point edgePos = pos + Point(.5 * size.X(), size.Y());
+	Point bottomOff(-26., .5 * bottom->Height() - 1);
```

Comment:
```suggestion
	Point bottomOff(-23.5, .5 * bottom->Height() - 1);
```
Bottom edge was misaligned again, this should line it up.

Reply:
Oh right I changed the width and forgot to adjust this